src/components/Notifications/Recipient.tsx

```diff
@@ -27,6 +27,9 @@ const greyColorName = style({
 const CommaSeparator = () => <span>, </span>;
 
 export const Recipient: React.FunctionComponent<RecipientProps> = (props) => {
+
+    const [ ignorePref ] = React.useState<EmailSystemProperties>();
```

Comment:
I think you still need to load this value into the `Action`.

Here is how to do it:

1. Update [NotificationAdapter](https://github.com/josejulio/notifications-frontend/blob/f711c5d03c6f66372489b3b44c9355cbc1b2fc5b/src/types/adapters/NotificationAdapter.ts#L35) to use the ignore preferences variable (`integration.ignorePreferences`) and pass it to `NotificationUserRecipient`
2. Update [NotificationUserRecipient](https://github.com/josejulio/notifications-frontend/blob/ffc76263cfc72f160bc31f693c3ffbeaa5d395b2/src/types/Recipient.ts#L56) to receive a third boolean for ignorePreferences, and assign it accordingly. You will need to update other places where it's called.
3.  The setting is per recipient. i.e. the recipient "all users" might not be forced but "send to admins" might be forced so I suggest to update:
`Users: {join(users.map(u => u.displayName), CommaSeparator)}` to something like:
```
Users: {join(users.map(u => <>
  {u.displayName}
  {u.ignoreUserPreferences && <span className={ greyColorName }> { props.action.integration.name }
                        <Tooltip content="This integration has been disabled. This action wil not fire until it is enabled." position="bottom">
                            <BanIcon className={ disabledLabelClassName } />
                        </Tooltip>
                    </span>)
</>), CommaSeparator)}
```
The lock icon should show only for the users that are forced.

Alternatively you could display it on the end by checking if there is at least one forced recipient:
```
const ignoreUserPreferences = users.some(u => u.ignoreUserPreferences)
```

4. Theorically the groups could also be forced, but there is no way to set it right now.